### PR TITLE
Committee endpoint

### PR DESCRIPTION
**Pull request by adborden**
Adds a committee endpoint in order to get information about a specific committee.

### CODE REVIEW

File: calculators/referendum_supporters_calculator.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 class ReferendumSupportersCalculator
-  def initialize(candidates: [], ballot_measures: [])
+  def initialize(candidates: [], ballot_measures: [], committees: [])
     @ballot_measures = ballot_measures
+    @committees_by_filer_id = committees
+      committees.where('"Filer_ID" IS NOT NULL').index_by { |c| c.Filer_ID }
```
Comment (tdooner) [inline calculators/referendum_supporters_calculator.rb]: This looks wrong to me, I don't think the WHERE will be applied to the original query.

You should remove the `committees` on line 5

Comment (adborden) [inline calculators/referendum_supporters_calculator.rb]: Oh yeah, thanks. Totally a typo.

**Comment by adborden** [comment]
So fixing that caused other problems, so it obviously wasn't working. Do I have this right? The committee files schedule E for their expenditures, that actually makes them the "payer" not the "payee", right?

**Comment by adborden** [comment]
Decided to dig into why we still had no IDs for some of the Oakland committees so turned up another fix. Looks like we were able to match a committee to all of the referendum expenditures in oakland :trophy: 

```
$ grep -r "id.*null" build/referendum/ | wc -l
0
```

**Comment by tdooner** [comment]
Ah yeah, this looks right. Filer_ID should always be a string. Thanks for digging in to find the bug. Next time, you should run `make process` and include the changes in `build` to the PR (in a separate commit from the rest of your code changes). 👍

**Pull request merged**